doc/related_projects.rst

```diff
@@ -219,4 +245,39 @@ Domain specific packages
 Snippets and tidbits
 ---------------------
 
-The `wiki <https://github.com/scikit-learn/scikit-learn/wiki/Third-party-projects-and-code-snippets>`_ has more!
+**Gists**
```

Comment:
But since we are going to include this heading anyways, it might be better to retain the gists in this page and save the trouble of maintaining another wiki page separately?